.github/workflows/lint-markdown-links-daily.yml

```diff
@@ -16,3 +16,23 @@ jobs:
           config-file: ".mlc.config.json"
           folder-path: "."
           max-depth: -1
+      - name: Send Notification to Slack
+        if: ${{ failure() }}
+        uses: slackapi/slack-github-action@v1.24.0
```

Comment:
is it on allowed github actions list? https://github.com/kyma-project/community/blob/main/docs/contributing/assets/allowed_actions.json 

it's recommended to use hash version instead of numeric tag

Reply:
Fixed.